### ISSUE TITLE
MA0144 - Not applicable for netstandard targets

### ISSUE THREAD

**Issue by JeevanJames** [meziantou/Meziantou.Analyzer#654]
### Version of the Meziantou.Analyzer NuGet package

2.0.113

### Rule Identifier

MA0144

### Target Framework

netstandard2.0

### C# Language version

C# 12

### Description

The analyzer is flagging this issue even on `netstandard2.0`, where the `OperatingSystem` class does not have any of the `IsXXXX` static methods like `IsWindows()`.

The Microsoft docs say that these methods are only available from .NET 5 and above.

While I can always suppress the issue, is there any way to not do the check when the runtime is below .NET 5?

### Reproduction Steps

Minimal code:

```c#
if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
{
    // Windows-specific code here
}
```

### Other information

The issue was not happening with 2.0.112. It only started once I updated to 2.0.113.

**Comment by meziantou** [comment]
The current detection of `OperatingSystem` is not good enough, that's why it's reported on older targets. #655 should fix it
